Application/Dtos/Blog/FeedPostVm.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Application.Dtos.Blog
+{
+    public class FeedPostVm
+    {
+        public Guid Id { get; set; }
+        public string Preview { get; set; }
+        public string Title { get; set; }
+        public DateTime CreatedOn { get; set; }
+        public string UserName { get; set; }
```

Comment:
I dont understend your propose. 
UserName is needed to display the name on the front

Reply:
Instead of keeping a part of User Entity (UserName) keep a reference to that Entity(UserID) that will allow you to be more flexible.
What if you will have to display not just a username but also his avatar? Or his account stats (posts created, comments created)? You will have to add everything into the FeedPostVm